src/test/java/org/sagebionetworks/web/unitserver/filter/CrawlFilterTest.java

```diff
@@ -114,6 +120,17 @@ public void testSynapseEntityPage() throws ServletException, IOException {
     assertTrue(outputString.contains(synapseID));
     assertTrue(outputString.contains(entityName));
     assertFalse(outputString.contains(META_ROBOTS_NOINDEX));
+
+    // verify we are asking for all entity types, except link
+    verify(mockSynapseClient)
+      .getEntityChildren(entityChildrenRequestCaptor.capture());
+    EntityChildrenRequest entityChildrenRequest = entityChildrenRequestCaptor.getValue();
+    List<EntityType> entityTypes = entityChildrenRequest.getIncludeTypes();
+    for (EntityType type : EntityType.values()) {
+      if (type != EntityType.link) {
+        assertTrue(entityTypes.contains(type));
+      }
```

Comment:
Could do this to check link as well
```suggestion
      assertEquals(EntityType.link != type, entityTypes.contains(type));
```